tasks/main.yml

```diff
@@ -1,27 +1,100 @@
-# SPDX-License-Identifier: MIT
 ---
-- name: Set platform/version specific variables
-  include_tasks: tasks/set_vars.yml
-
-# Examples of some tasks:
-- name: Ensure required packages are installed
+- name: "Ensure sudo is installed"
   package:
-    name: "{{ __template_packages }}"
-    state: present
-    use: "{{ (__template_is_ostree | d(false)) |
-             ternary('ansible.posix.rhel_rpm_ostree', omit) }}"
-
-- name: Ensure required services are enabled and started
-  service:
-    name: "{{ item }}"
-    state: started
-    enabled: true
-  loop: "{{ __template_services }}"
-
-- name: Generate /etc/{{ __template_foo_config }}
+    name: "sudo"
+    state: "present"
+  retries: 3
+  register: "sudo_installed"
+  until: "sudo_installed is succeeded"
```

Comment:
I don't think we need retries for installing the package. Remove `retries` and `until`